packages/server/package.json

```diff
@@ -35,6 +35,7 @@
     "@nestjs/platform-express": "^9.0.0",
     "@prisma/client": "4.10.1",
     "apollo-server-express": "^3.11.1",
+    "class-transformer": "^0.5.1",
```

Comment:
Is this package used?

Reply:
no, but when I was trying to run nest start, it gave an error and asked me to install this package. But right now I tried to uninstall it, it works fine.